experiments/pagination/README.md

```diff
@@ -0,0 +1,98 @@
+on pagination fields.
+
+## Problem
+At some point, the optimal query has been created, using all the performance enhancements, but still produces a lot
+of hits and takes a long time to complete. Such a delay can be a bad experience for the user and uses a lot of 
+system resources. 
+
+## Goals
+This experiment tries to show the power of Pagination and the use of [Querybuilder](https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/com/day/cq/search/QueryBuilder.html)
+properties.  A customer’s search may include introduction of their own custom APIs, and not have the UI call
+Querybuilder directly.  That is fine, as this experiment simply tries to illustrate the principles of pagination. 
+
+## Non-Goals
+It does not try to teach how to:
+ * create a beautiful web page
+ * script perfect JS/JQuery
+ * optimize a search for best performance
+ * validate field values
+ * etc.
+
+## Setup
+Tools to Install:
+* Maven (3.6.3 or more recent)
+
+_NOTE_: An author instance running on port 4502 is required for this experiment.
+
+We'll begin by building and installing the package included in this project.
+
+1. Clone this Git repo to a local location
+1. Using a console, go to the `aem-project` folder located in the project root (created from the AEM Project Archetype).
+1. Successfully build and install the package to your localhost's author instance by running `mvn -PautoInstallSinglePackage clean install`
+1. To verify the installation, open Pager: http://localhost:4502/content/searchtester/us/en/experiments/pager.html
+
+## Pager 
+On the [page](http://localhost:4502/content/searchtester/us/en/experiments/pager.html) you will see 4 fields which
+you may recognize as
+[Querybuilder](https://helpx.adobe.com/experience-manager/6-5/sites/developing/using/reference-materials/javadoc/com/day/cq/search/QueryBuilder.html)
+fields:
+
+* limit: Page Size
+* guessTotal: Whether to finish the full query to determine the total size (see the [Guess Total experiment](https://github.com/adobe/aem-search-experiments/tree/master/experiments/large-result-sets) for more information)
+* offset: Where, in the result list, to start extracting hits to return 
+* path: The base path where the search will begin
+
+After clicking _Load Page_, these fields will be updated, if required, in order to be ready for the next load.
+
+You will see 2 buttons:
+
+* Load Page: Use the fields' values to start a search of cq:Page items
+* Reset Fields: Reset all the values to their default values
+
+You will see 2 text area fields:
+
+* Results: The list of all the hits already returned with the most recent hits appended to the bottom. Scrolling to the bottom of the list will trigger the next load (i.e. infinite scroll).
+* Details: The details of the most recently executed search call including time duration, URL and the response.
+
+## Test #1: 
```

Comment:
From @clotton 
I'd think, usually, the 1st test is "worst case scenario" and the following tests show improvement on that result.  I noticed the UI starts with a p.limit of 10.  I would have thought Test 1 would open the floodgates and be the longest possible duration and biggest result set to make the improvements more dramatic.